dash_bio/Molecule3dViewer.py

```diff
@@ -13,12 +13,15 @@ class Molecule3dViewer(Component):
 
 Keyword arguments:
 - id (string; optional): The ID used to identify this component in callbacks
-- selectionType (a value equal to: 'atom', 'residue', 'chain'; optional): The selection type - may be atom, residue or chain
-- backgroundColor (string; optional): Property to change the background color of the molecule viewer
-- backgroundOpacity (number; optional): Property to change the backgroun opacity - ranges from 0 to 1
-- styles (list; optional): Property that can be used to change the representation of
-the molecule. Options include sticks, cartoon and sphere
-- modelData (optional): The data that will be used to display the molecule in 3D
+- selectionType (a value equal to: 'atom', 'residue', 'chain'; default 'atom'): The selection type - may be atom, residue or chain
+- backgroundColor (string; default '#FFFFFF'): Property to change the background color of the molecule viewer
+- backgroundOpacity (number; default 0): Property to change the backgroun opacity - ranges from 0 to 1
+- styles (dict; optional): Property that can be used to change the representation of
+the molecule. Options include sticks, cartoon and sphere. styles has the following type: list of dicts containing keys 'color', 'visualization_type'.
+Those keys have the following types:
```

Comment:
```suggestion
These keys have the following types:
```

Reply:
This also needs to go in https://github.com/plotly/dash/blob/fbb2afcc61395abd5a3d63408f4dfddc6305bebd/dash/development/_py_components_generation.py#L479.

Reply:
Oh, that's where it's coming from. Thanks for pointing me to the source.